### PR TITLE
feat: added home page for mini app

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
- added light dark toggle
  
image

image

### CODE REVIEW

File: src/components/home/home.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.header_logo {
+    font-size: 1.5rem;
+    font-weight: 400;
+    margin-right: auto;
+}
+
+.dark_light-icon {
+    color: white;
```
Comment (t007rushi) [inline src/components/home/home.css]: color variable

**Review by t007rushi — APPROVED** [review]
Good work 
naming is also fine
go ahead

**Review by paarthbajaj — APPROVED** [review]
I liked the transition really. That was smooth. Code is clean. I will suggest you to make some more utility classes to avoid code repetition (for ex. cursor: pointer, etc.). Great efforts. You can merge.

**Review by Kothalkarpriya — APPROVED** [review]
Good Start! You can merge the changes

**Pull request merged**